chainer/functions/vae.py

```diff
@@ -0,0 +1,106 @@
+import math
+
+import chainer.functions as F
+from chainer import variable
+
+
+def gaussian_kl_divergence(z_mean, z_ln_var):
```

Comment:
If we separate this function from the context of VAE, users of this function can be confused what `z` in `z_mean` and `z_ln_var` stands for. So `mean` and `ln_var` is enough as names of arguments.